docs/field_types.md

```diff
@@ -319,3 +319,118 @@ $field->setOptions([
     // Your options here
 ]);
 ```
+
+Callable
+--------
+
+The Callable column aims to offer almost as much flexibility as the Twig column, but without requiring the creation of a template.
```

Comment:
For the documentation, please open a PR on the Sylius stack documentation.

Reply:
Done, see https://github.com/Sylius/Stack/pull/228

Reply:
So, you should revert the doc part from this PR ;)